.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,6 +18,11 @@ Describe the problem with the model and the results you are seeing.
 What environment are you using?
```

Comment:
separate this out into system information?
### System Information
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- ONNX version:
- backend/runtime being used: (e.g. onnxruntime, pytorch):